docs/src/siteConfig.js

```diff
@@ -5,9 +5,10 @@ export const siteConfig = {
   npmUrl: 'https://www.npmjs.com/package/@hyperjumptech/monika',
   repoUrl: 'https://github.com/hyperjumptech/monika',
   algolia: {
-    appId: '',
-    apiKey: '',
-    indexName: '', // algoliaOptions: {
+    appId: '5O2YZGK4NG',
+    apiKey: '45f684bf6230e7980f10b4a9209fce83',
```

Comment:
Don't hard code these. Read the value from the environment variable `ALGOLIA_APPLICATION_ID` and `ALGOLIA_SEARCH_API_KEY `

Reply:
okay, i created the environment